deploy/olm-catalog/service-telemetry-operator/manifests/service-telemetry-operator.clusterserviceversion.yaml

```diff
@@ -353,12 +353,18 @@ spec:
           - statefulsets
           verbs:
           - '*'
+        - apiGroups:
+          - cert-manager.io
+          resources:
+          - issuers
+          - certificates
+          verbs:
+          - '*'
```

Comment:
This block is generated / updated in the CSV as sourced from `deploy/role.yaml` via `operator-sdk`. This is not something that should be manually applied by hand.

```
operator-sdk generate bundle --channels unstable --default-channel --manifests --metadata
```